experimental/generation/generator/packages/library/templates/standard/en-us/form.en-us.lg

```diff
@@ -19,9 +19,8 @@
 > Display value for a property value
 - ${template(`${property}_Value`, val)}
 
-# entityValue(property, val)
-> Display value for an entity
-- ${template(`${property}Entity_Value`, val)}
+# entityValue(entity)
+- ${value(entity.value)}
```

Comment:
Include comment in-between, i.e.
# entityValue(entity)
> Display value for an entity
- template('${entity.type}_Value', entity)